sys/sys/sysctl.h

```diff
@@ -163,7 +163,9 @@ struct ctlname {
     (defined(__mips__) && defined(__mips_n64))
```

Comment:
So I wonder why this isn't just `COMPAT_FREEBSD32`?  Would be nice to fix that upstream.  If it doesn't work upstream, then we may need to change `SCTL_MASK64` to have a similar list of architecture #if clauses.